docs/guides/nwaku/run-docker.md

```diff
@@ -45,7 +45,7 @@ docker run [OPTIONS] [IMAGE] [ARG...]
 Run `nwaku` using the most typical configuration:
 
 ```shell
-docker run -i -t -p 60000:60000 -p 9000:9000/udp statusteam/nim-waku:v0.20.0 \
+docker run -i -t -p 60000:60000 -p 9000:9000/udp wakuorg/nwaku:v0.32.0 \
```

Comment:
Yes, here we would need a generic tag on docker hub also (as we had it before on harbor), like `latest-release` or so.